incubator/hnc/pkg/kubectl/create.go

```diff
@@ -36,9 +38,20 @@ var createCmd = &cobra.Command{
 		// another namespace, this will be caught by the admission controller.
 		updates := hcUpdates{requiredChildren: args}
 		updateHC(client, updates, parent)
+		// Create hns instances.
+		createHNS(client, parent, args)
 	},
 }
 
+func createHNS(cl Client, ns string, nhnss []string) {
+	for _, nhns := range nhnss {
+		hns := &api.HierarchicalNamespace{}
+		hns.Name = nhns
+		hns.Namespace = ns
+		cl.createHierarchicalNamespace(hns)
```

Comment:
Since we'll never modify any properties other than the name and ns, maybe the client should just take those explicitly and do everything internally? Eg:

`cl.createHNS(parentNS, childNS)`